### PR TITLE
URL Checker enhancements

### PR DESCRIPTION
**Pull request by ACueva**

### CODE REVIEW

**Comment by ACueva** [comment]
@csmoore  Please review.

**Comment by csmoore** [comment]
I tested this & it looks good/works as expected to me

@elinz - just let me know if you would also prefer to review & do the merge - If I don't hear back, I'll merge in 2 days.

File: SupportFiles/UrlChecker.py [hunk 1]
```diff
@@ -84,66 +153,71 @@ def main():
...
-		else : 						
-			if urlCount == 0 :			
+								badUrl, code, message = checkUrl(url)
+
+								if badUrl or verbose:
+									urlCount += 1
+									if newUrlFile:
+										urlFileCount += 1
+										newUrlFile = False
+									urlError = [str(urlCount), str(urlFileCount), currentFile, currentFile, str(lineNumber), url, code, message]
```
Comment (csmoore) [inline SupportFiles/UrlChecker.py]: @ACueva - I just noticed this in the output csv, but in your pylinting splendor, you repeated "currentFile" twice here in error, this should be fixed before PR is merged

**Comment by ACueva** [comment]
Good catch @csmoore

**Pull request merged**